scio-elasticsearch/src/main/scala/com/spotify/scio/elasticsearch/package.scala

```diff
@@ -82,13 +74,13 @@ package object elasticsearch {
      * @param f function to transform arbitrary type T to Elasticsearch [[ActionRequest]]
      * @param numOfShard number of parallel writes to be performed, recommended setting is the
      *                   number of pipeline workers
-     * @param errorHandler function to handle error when performing Elasticsearch bulk writes
+     * @param e function to handle error when performing Elasticsearch bulk writes
```

Comment:
I actually prefer `errorHandler` or `errorFn`, but `e` in public API is probably a little bit too cryptic.